plugin/mpi/PETSc-code.hpp

```diff
@@ -4705,10 +4731,19 @@ namespace PETSc {
           if (op) {
             ffassert(op);
             const OneOperator* codeM =
+              op->Find("(", ArrayOfaType(atype< long >( ), atype< double >( ),
+                                          atype< KN< PetscScalar >* >( ), false));
+            user->mon = new NonlinearSolver< Type >::IMonF_O(in->n, stack, codeM);
+            SNESMonitorSet(snes, Monitor< NonlinearSolver< Type > >, &user, NULL);
+          }
+          op = nargs[12] ? dynamic_cast< const Polymorphic* >(nargs[12]) : nullptr;
+          if (op) {
+            ffassert(op);
+            const OneOperator* codeC =
               op->Find("(", ArrayOfaType(atype< long >( ), atype< double >( ), atype< double >( ),
-                                         atype< double >( ), atype< KN< PetscScalar >* >( ),
-                                         atype< KN< PetscScalar >* >( ), false));
-            user->conv = new NonlinearSolver< Type >::IConvF_O(in->n, stack, codeM);
+                                          atype< double >( ), atype< KN< PetscScalar >* >( ),
+                                          atype< KN< PetscScalar >* >( ), false));
```

Comment:
```suggestion
                                         atype< double >( ), atype< KN< PetscScalar >* >( ),
                                         atype< KN< PetscScalar >* >( ), false));
```
Unneeded whitespace changes, then I think we are good to go.